test/edu-benefits/5490/config/benefitSelection.unit.spec.jsx

```diff
@@ -24,19 +35,64 @@ describe('Edu 5490 benefitSelection', () => {
       .to.equal(5);
   });
 
-  it('should show inline message', () => {
+  it('should render inputs for spouse relationship', () => {
     const form = ReactTestUtils.renderIntoDocument(
       <DefinitionTester
           schema={schema}
-          data={{ benefit: 'chapter35' }}
+          data={{}}
+          state={{
+            benefitSelection: {
+              uiSchema: {
+                benefit: {
+                  'ui:options': {
+                  }
+                }
+              }
+            }
+          }}
+          formData={{ relationship: 'spouse' }}
+          uiSchema={uiSchema}/>
+    );
+    const fields = ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'input').concat(
+      ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'select')
+    );
+
+    expect(fields.length)
+      .to.equal(2);
+  });
+
+  it('should show nested content', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          data={{}}
+          state={{
+            benefitSelection: {
+              uiSchema: {
+                benefit: {
+                  'ui:options': {
+                  }
+                }
+              }
+            }
+          }}
           uiSchema={uiSchema}/>
     );
     const formDOM = findDOMNode(form);
 
-    // check that an alert box is shown below the chapter35 option
-    expect(formDOM
-      .querySelector('.form-radio-buttons + span .usa-alert')
-    ).not.to.be.null;
+    expect(formDOM.querySelector('.form-radio-buttons + span .usa-alert')).to.be.null;
+    ReactTestUtils.Simulate.change(formDOM.querySelector('#root_benefit_0'), {
+      target: {
+        checked: true
+      }
+    });
+    // check that an alert box is shown when an option is selected
```

Comment:
I think it's good to leave in here for now as a reminder that this is something we need to loop back to. We can then search in the unit tests for `setTimeout` to find where we need to fix up once we have a solution.